zipkin-storage/cassandra3/src/main/java/zipkin/storage/cassandra3/DefaultSessionFactory.java

```diff
@@ -123,6 +125,13 @@ static Cluster buildCluster(Cassandra3Storage cassandra) {
     builder.withPoolingOptions(new PoolingOptions().setMaxConnectionsPerHost(
         HostDistance.LOCAL, cassandra.maxConnections
     ));
+
+    builder.withQueryOptions(
```

Comment:
this is the only one I don't know just by reading it. Can you put a comment line of why we do this, or tell me and I'll add it? Thx!

Reply:
if `local_dc` isn't defined then LOCAL_ONE sticks requests to the first seed host that the driver connects to.
ONE will at least in this situation connect to any dc, and then any latency awareness will bring it back to (eventually) using the local dc.

furthermore `LOCAL_ONE` can fail easily if you're not using 'NetworkTopologyStrategy` but that's a fix for another day.

Reply:
added comment. (can you please squash the 4 commits!)

Reply:
great info. ty